doc/api/util.md

```diff
@@ -423,6 +426,11 @@ changes:
     `object`. This is useful to minimize the inspection output for large
     complicated objects. To make it recurse indefinitely pass `null` or
     `Infinity`. **Default:** `Infinity`.
+  * `maxClockTime` {number} Specifies the maximum time in milliseconds that
```

Comment:
Just `maxTime`?  I'm sure everyone will understand that to be wall clock time, i.e., human time.

`maxClockTime` inevitably raises the question "what clock?"